iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/list.inl

```diff
@@ -701,7 +702,7 @@ inline bool list<T, Capacity>::isInvalidIterOrDifferentLists(const const_iterato
 template <typename T, uint64_t Capacity>
 inline void list<T, Capacity>::errorMessage(const char* source, const char* msg) noexcept
 {
-    std::cerr << source << " ::: " << msg << std::endl;
+    IOX_LOG(ERROR) << source << " ::: " << msg;
```

Comment:
Since `errorMessage` is called in 'normal' situations like the list is empty/full when an element shall be erased/added, I think `INFO` would be sufficient here as log level.